students/jeff_shabani/session03/mailroom.py

```diff
@@ -0,0 +1,71 @@
+#!/usr/bin/env python3
+import sys
+
+from operator import itemgetter
+from collections import OrderedDict
+
+DONORS = {'William B': [120, 130, 50],
+          'Sammy Maudlin': [500, 125, 670, 1000],
+          'Bobby Bittman': [10],
+          'Skip Bittman': {75, 125, 19},
+          'Ashley Lashbrooke': [10000, 15000]}
+
+prompt = "\n".join(("Welcome to my charity!",
+          "Please select and option below:",
+          "1 - Send a Thank You",
+          "2 - Create a Report",
+          "3 - Quit",
+          ">>> "))
+
+def view_donor_names():
+    for k,v in DONORS.items():
+        print(k)
+
+def add_new_donor(name, donor_list):
+    amount = int(input(f'How much would this donor like to donate?'))
+    donor_list[name] = [amount]
+
+
+
+def get_a_name():
+    answer = input('Please enter a donor Full Name.')
+    if answer.lower() == 'list':
+        view_donor_names()
+        answer = input('Please enter a donor Full Name.')
+    amount = int(input('How much would this donor like to donate?'))
+    if answer not in DONORS:
+        DONORS[answer] = [amount]
+    else:
+        for k,v in DONORS.items():
+            if k.lower() == answer.lower():
+                v.append(amount)
+    print(f'Thank you {answer.split()[0]} for you generous donation of ${amount:,.0f}')
+    print()
```

Comment:
So I re wrote it a bit, also notice the spacing between the lines makes it more readable instead of text on every line
```
def add_donations_and_send_thank_you():

    while True:

        answer = input('Please enter a donor Full Name.')

        if answer.lower() == 'list':
            view_donor_names()
            continue

        amount = int(input('How much would this donor like to donate?'))

        if answer not in DONORS:
            DONORS[answer] = [ammount]

        else:
            for name, donations in DONORS.items():
                if name == answer:
                    donations.append(amount)

        print(f'\nThank you {answer.split()[0]} for you generous donation of ${amount:,.0f}\n')
        break
```